src/main/java/org/fcrepo/importexport/common/Config.java

```diff
@@ -386,28 +405,29 @@ public static Config fromFile(final Map<String, String> configVars) throws Parse
             } else if (entry.getKey().equalsIgnoreCase("rdfLang")) {
                 c.setRdfLanguage(entry.getValue());
             } else if (entry.getKey().trim().equalsIgnoreCase("binaries")) {
-                if (entry.getValue().equalsIgnoreCase("true") || entry.getValue().equalsIgnoreCase("false")) {
-                    c.setIncludeBinaries(Boolean.parseBoolean(entry.getValue()));
-                } else {
-                    throw new ParseException(String.format(
-                        "binaries configuration parameter only accepts \"true\" or \"false\", \"{}\" received",
-                        entry.getValue()), lineNumber);
-                }
+                c.setIncludeBinaries(parseBoolean(entry, "binaries", lineNumber));
+            } else if (entry.getKey().trim().equalsIgnoreCase("overwriteTombstones")) {
+                c.setIncludeBinaries(parseBoolean(entry, "overwriteTombstones", lineNumber));
             } else if (entry.getKey().trim().equalsIgnoreCase("external")) {
-                if (entry.getValue().equalsIgnoreCase("true") || entry.getValue().equalsIgnoreCase("false")) {
-                    c.setRetrieveExternal(Boolean.parseBoolean(entry.getValue()));
-                } else {
-                    throw new ParseException(String.format(
-                        "external configuration parameter only accepts \"true\" or \"false\", \"{}\" received",
-                        entry.getValue()), lineNumber);
-                }
+                c.setIncludeBinaries(parseBoolean(entry, "external", lineNumber));
```

Comment:
It looks like these errors were not caught by unit tests because (1) there was no test and (2) this is dead code duplicating the parsing in ArgParser.  I'll remove this code and add a test for the ArgParser version.